### ISSUE TITLE
DSpace 7: Page jumps during submission

### ISSUE THREAD

**Issue by f-rapp** [DSpace/DSpace#9283]
**Describe the bug**
During submission, the page sometimes "jumps" and users have to navigate back to where they were before the jump (scroll up/down). We noticed the page jumps occur when DSpace caches the submission (the caching is visible at the bottom of the page).

**To Reproduce**
Start a submission and fill in the fields. At some point when DSpace caches, the page "jumps". It may be that there are additional factors which need to be in place for the page to jump. We have heard from other institutions that face the same problem and do not have a solution for this issue so far.

**Expected behavior**
When DSpace caches a submission, the page does not "jump".

**Related work**
Link to any related tickets or PRs here.

**Comment by alanorth** [comment]
@f-rapp I've noticed this too. Do you have type binding enabled in your submission form by any chance?

Some other factors could be: when you have more than one submission step. We have three, and DSpace attempts to auto save when the users moves between them, and the page jumps to the bottom.

**Comment by f-rapp** [comment]
@alanorth Yes, we use type binding in our submission form. We also have more than one submission step.

**Comment by alanorth** [comment]
I wasn't able to reproduce this with the default configuration on demo.dspace.org, but I was able to reproduce this on our repository with type binding disabled—so it's not that. But there is definitely something about moving between submission steps, as you can see it jumps twice here towards the end when I move between "Describe" steps:

[recording.webm](https://github.com/DSpace/DSpace/assets/191754/e264357a-54c0-4842-8f37-089f2beafd31)

(Video doesn't play in Firefox for some reason that is beyond me, but should on Chrome).

Also notice how the auto save causes focus to be lost when you click to enter the year in the date issued field.
